freqtrade/rpc/external_message_consumer.py

```diff
@@ -205,11 +205,6 @@ async def _create_connection(self, producer: Dict[str, Any], lock: asyncio.Lock)
                     # Now receive data, if none is within the time limit, ping
                     await self._receive_messages(channel, producer, lock)
 
-            # Catch invalid ws_url, and break the loop
-            except websockets.exceptions.InvalidURI as e:
-                logger.error(f"{ws_url} is an invalid WebSocket URL - {e}")
-                break
-
```

Comment:
I don't think it's safe to omit this. while we now provide host:port separately, there's no additional checks in place for this.

i could therefore supply a port > 66000 (or negative port) - or an invalid hostname (one that doesn't produce a valid url/ip)

```
"host": "127.0.0.266",
"port": 6666000
```

while we'll not crash - it'll cause a loop and have the bot refuse to properly start.

Reply:
So providing a bad port causes a ValueError, and providing an invalid hostname causes a socket.gaierror

Reply:
That's correct - problem is - valueError is looping "forever" , trying to reconnect several times per second (spamming the logs, effectively).

We can counter this by limiting the range in the schema though (0-65535) (we "continue" this - even though we really have no way to process further).